diff -rup lohit-devanagari-2.4.3/Lohit-Devanagari.sfd lohit-devanagari-2.4.3_mod/Lohit-Devanagari.sfd
--- lohit-devanagari-2.4.3/Lohit-Devanagari.sfd	2010-02-04 13:49:05.000000000 +0530
+++ lohit-devanagari-2.4.3_mod/Lohit-Devanagari.sfd	2011-01-24 15:41:36.419383853 +0530
@@ -13,13 +13,14 @@ Descent: 205
 LayerCount: 2
 Layer: 0 1 "Back"  1
 Layer: 1 1 "Fore"  0
+NeedsXUIDChange: 1
 XUID: [1021 808 1214965479 5361592]
 FSType: 0
 OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1159459513
-ModificationTime: 1264743564
+ModificationTime: 1295863721
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -902,7 +903,7 @@ NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 2257 37 13
+WinInfo: 2146 37 13
 GridOrder2: 1
 Grid
 -13 606 m 1,0,-1
@@ -950,7 +951,7 @@ Grid
 EndSplineSet
 TeXData: 1 0 0 323584 161792 107861 0 1048576 107861 783286 444596 497025 792723 393216 433062 380633 303038 157286 324010 404750 52429 2506097 1059062 262144
 AnchorClass2: "Anchor-0"  "'abvm' Above Base Mark in Devanagari lookup 3" "Anchor-1"  "'abvm' Above Base Mark in Devanagari lookup 2" "Anchor-2"  "'abvm' Above Base Mark in Devanagari lookup 1" "Anchor-3"  "'blwm' Below Base Mark in Devanagari lookup 0" 
-BeginChars: 65751 352
+BeginChars: 65751 353
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -29568,5 +29569,37 @@ SplineSet
  40 588 l 1,0,-1
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 352
+Width: 609
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 500 m 0,0,1
+ 359 509 359 509 359 514 c 0,2,-1
+ 59 514 l 0,3,-1
+ 85 577 l 0,4,-1
+ 346 577 l 0,5,6
+ 340 589 340 589 331 599 c 0,7,8
+ 280 654 280 654 165 654 c 0,9,-1
+ 59 654 l 0,10,-1
+ 85 717 l 0,11,-1
+ 583 717 l 0,12,-1
+ 557 654 l 0,13,-1
+ 367 654 l 0,14,15
+ 406 631 406 631 420 577 c 0,16,-1
+ 583 577 l 0,17,-1
+ 557 514 l 0,18,-1
+ 429 514 l 0,19,20
+ 429 358.613 429 358.613 171 340 c 0,21,-1
+ 507 0 l 0,22,-1
+ 407 0 l 0,23,-1
+ 70 337 l 0,24,-1
+ 70 396 l 0,25,26
+ 345.891 394 345.891 394 358 500 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
